stack.yaml

```diff
@@ -21,7 +21,7 @@ extra-deps:
   - git:
       https://gitlab.com/morley-framework/morley.git # CI can't use SSH
     commit:
-      2d0506578493fec3851d711a5cc26c9dc5885001 # morley-1.7.0
+      619915ceccb66ba36d0eca06cedd76f605a15bd3 # Unreleased: rvem/#334-expect-custom-error-in-nettest
```

Comment:
It's already merged to master so this comment looks confusing. This branch doesn't even exist, just write `master` here.